apps/admin-gui/src/app/admin/pages/admin-page/admin-services/service-detail-page/service-detail-page.component.html

```diff
@@ -40,11 +40,20 @@
         <span>
           {{service.enabled ? ('SERVICE_DETAIL.ENABLED' | translate) : ('SERVICE_DETAIL.DISABLED' | translate)}}
           <button (click)="changeServiceStatus()" mat-icon-button>
-            <mat-icon class="align-content-center">
+            <mat-icon class="mr-2 mb-1">
               {{service.enabled ? "check_circle_outline" : "block"}}
             </mat-icon>
           </button>
         </span>
+        {{'SERVICE_DETAIL.EXPIRED_MEMBERS' | translate}}:
+        <span>
+          {{service.useExpiredMembers ? ('SERVICE_DETAIL.PROPAGATED' | translate) : ('SERVICE_DETAIL.NOT_PROPAGATED' | translate)}}
+          <button (click)="changeServiceExpiredMembersUsage()" mat-icon-button>
+            <mat-icon class="mr-2 mb-1">
+              {{service.useExpiredMembers ? "filter_alt_off" : "filter_alt"}}
+            </mat-icon>
+          </button>
+        </span>
```

Comment:
```suggestion
            <button (click)="changeServiceStatus()" class="icon-button" mat-icon-button>
            <mat-icon class="align-bottom">
              {{service.enabled ? "check_circle_outline" : "block"}}
            </mat-icon>
          </button>
        </span>
        , {{'SERVICE_DETAIL.EXPIRED_MEMBERS' | translate}}:
        <span>
          {{service.useExpiredMembers ? ('SERVICE_DETAIL.PROPAGATED' | translate) : ('SERVICE_DETAIL.NOT_PROPAGATED' | translate)}}
          <button (click)="changeServiceExpiredMembersUsage()" class="icon-button" mat-icon-button>
            <mat-icon class="align-bottom">
              {{service.useExpiredMembers ? "filter_alt" : "filter_alt_off"}}
            </mat-icon>
          </button>
        </span>
        <span>
          , {{'SERVICE_DETAIL.EVALUATE_CONSENTS' | translate}}:
          <button (click)="evaluateConsents()" class="icon-button" mat-icon-button>
            <mat-icon class="align-bottom"> swipe_right_alt </mat-icon>
          </button>
        </span>
```
The suggestion is not 1 to 1 to the row
What about this? Does it look a little bit better? What do you think?

into the CSS file:
`.icon-button {
  line-height: inherit !important;
  width: inherit !important;
  height: inherit !important;
}
`